src/main/resources/application-gramine.yml

```diff
@@ -0,0 +1,13 @@
+tee:
+  secret-provisioner:
+    web:
```

Comment:
Great introduction of application-gramine.yml & application-scone.yml!
To discuss, but for consistency I feel we should
1.  relocate `secret-provisioner` blocks to application.yml (and remove default values)
or
2.  move `workflow` parts from application.yml to application-gramine/scone.yml
Do you see my point?

Reply:
I see your point, sure! I prefer the second version as it provides an easier testing setup. Here is the commit: https://github.com/iExecBlockchainComputing/iexec-sms/pull/119/commits/4a641a68960de4981e7cb44bca9a3453194e09f7.

Reply:
Isn't it a `application-gramine-development.yml` instead?

Reply:
The eternal question...!